# Comparative Pan-genome Analysis of *Staphylococcus epidermidis*
## Population Structure and Functional Profiling

---

### 1. Background

Based on MLST analysis of 124 *S. epidermidis* RefSeq genomes, a total of 69 sequence types (STs) were identified.

Among the 69 sequence types, the most prevalent were **ST-2 (18 isolates)**, which is known to be associated with clinical pathogenicity, and **ST-5 (5 isolates)**, which is considered a relatively generalist type.

**Pan-genome analysis** was conducted using the **Roary**, with gene annotation data of the 124 strains obtained through **Prokka**.

Orthologous genes were clustered using a 95% protein sequence identity threshold, and genes present in more than 99% of all strains were defined as core genes.

| Category | Count | Percentage(%) | Definition |
| :--- | :---: | :---: | :--- |
| **Total Isolates** | **124** | - | Total number of strains included in the analysis |
| **Total Genes** | **10,014** | 100% | otal gene set identified across all strains |
| **Core Genes** | **1,522** | 15.2% | Core genes present in ≥99% of all strains |
| **Accessory Genes** | **8,492** | 84.8% | Variable genes present in only a subset of strains (cloud/shell genes) |

---

### 2. Goal

1.  **Genetic Distance:** The genetic differences between the major sequence types, ST-2 and ST-5, were examined using dendrogram constructed based on genomic composition dissimilarity
  
2.  **Comparative Genomics:** Identifying genetic markers that distinguish the pathogenic dominant type (ST-2) from the generalist type (ST-5)
  
3.  **Functional & Metabolic Profiling:** Analyzing gene-level functional classifications and metabolic pathways (Auxotrophy)

---

### 3. Tools

* **Genome Annotation:** Prokka
* **Pan-genome Analysis:** Roary (Core/Accessory gene definition)
* **Dendrogram:** ITOL (Based on gene presence/absence matrix)
* **GWAS-like Association:** Scoary (Identification of group-specific markers)
* **Functional Profiling:** Custom Python Scripting (Keyword screening based on Prokka annotation)
* **Auxotrophy Visualization:** Python & ITOL Overlay

---

### 4. Results

#### 4.1 Population Structure

Although MLST analysis identified 69 distinct sequence types, clustering based on the overall gene presence–absence patterns grouped all strains into two major lineages, which subsequently branched into several sublineages.

The genetic differences between ST-2 and ST-5 among the various MLST types were examined using a dendrogram.

<img width="768" height="776" alt="image" src="https://github.com/user-attachments/assets/b0841d0f-63be-423f-aae5-3feefa5cddb9" />


* **Cluster 1 (Red Clade):** **ST-2 lineage, centered on the pathogenic ST-2**.
* **Cluster 2 (Blue Clade):** **ST-5 lineage, centered on ST-5**.
* **Cluster 3 (Grey Clade):** **Lineages of the remaining 67 STs**.

Some strains classified as ST-5 by MLST were located within the ST-2 cluster, while some ST-2 strains were positioned far from the ST-2 cluster, indicating exceptional patterns.



#### 4.2 Comparative Genomics

**Statistically significant genes (diagnostic markers)** distinguishing the two groups were identified using **Scoary**.

* **ST-2 (Pathogenic Type):**
    * A highly significant marker gene, **group_4200 (P < 1.0×10⁻¹³, sensitivity 100%)**, was identified.
    * BLAST analysis: The gene encodes a protein derived from a **bacteriophage (*Caudoviricetes sp.*)**.
 
* **ST-5 (Commensal Type):**
    * No unique marker identified.


#### 4.3 Comprehensive Functional Profiling

Based on annotations generated by Prokka, functional analysis at the gene level was performed across seven categories.

| Functional Category | Gene Count | Proportion(%) | Biological Implication  |
| :--- | :---: | :---: | :--- |
| **Hypothetical / Unknown** | **6,543** | 65.34% | Uncharacterized genes with unknown functions |
| **Enzymes (Metabolism)** | **1,549** | 15.47% | Basic metabolic enzymes essential for survival and growth (e.g., kinases, synthases) |
| **Mobile Elements** | **348** | 3.47% | HGT-related genes driving horizontal gene transfer, including phages and transposons |
| **Transporter / Membrane** | **291** | 2.91% | Genes involved in nutrient acquisition, waste removal, and drug efflux pumps |
| **Translation / Ribosome** | **202** | 2.02% | Ribosomal and tRNA-related genes, mostly part of the core genome |
| **Regulation** | **197** | 1.97% | Factors controlling gene expression in response to environmental changes (e.g., repressors, regulators) |
| **Virulence / Resistance** | **90** | 0.90% | Genes directly related to host attack (toxins) or antibiotic resistance |
| **Other Function** | **794** | 7.92% | Other structural and accessory proteins |
| **Total** | **10,014** | 100.0% | - |



#### 4.4 Auxotrophy

Genes involved in the biosynthetic pathways of three essential amino acids—**arginine**, **valine**, and **leucine**—were analyzed.

* **Valine / Leucine:** No biosynthetic genes identified
* **Arginine:** Only three biosynthetic genes present

The number of arginine biosynthesis genes present in each of the 124 strains was examined using a dendrogram.

Auxotroph analysis revealed that the loss of certain genes was not limited to specific ST types but occurred across all 124 strains.


<img width="830" height="802" alt="image" src="https://github.com/user-attachments/assets/c712c038-3024-4338-9923-1c928417808e" />

